### PR TITLE
add players list

### LINKED ISSUE #8: minha-vez | add players list on lobby
## Introduction
We are show a list of people that have joined the lobby

## Implementation
1. Add list with:
i. player's position (listitem's index)
ii. the player name
iii. in case the item of the list is the current user, it should show `Você`
2. Add a `Em espera` title

### PR DESCRIPTION
**Pull request by frgiovanna**
### Motivation

Closes #8 

This PR adds players list on Lobby page

### Screenshot

Screenshot 2024-12-03 at 10 49 03

### CODE REVIEW

File: src/pages/Lobby/components/PlayersTable/PlayersTable.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Position, Row } from './styles';
+import { PlayersTableProps } from './types';
+
+export function PlayersTable({ players }: PlayersTableProps) {
+  function getPlayerNameItem(current: boolean, name: string) {
+    if (!current) {
+      return name;
+    }
+    return 'Você';
+  }
```
Comment (Wendlereis) [inline src/pages/Lobby/components/PlayersTable/PlayersTable.tsx:10]: **Suggestion:** this function can be defined outside the component as it does not need any state or prop

File: src/pages/Lobby/components/PlayersTable/PlayersTable.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <table>
+      <tbody>
```
Comment (Wendlereis) [inline src/pages/Lobby/components/PlayersTable/PlayersTable.tsx:14]: **Question:** Why did you choose to use the table?

Comment (frgiovanna) [reply src/pages/Lobby/components/PlayersTable/PlayersTable.tsx:14]: I wanted the first item (the position) to have a fixed width to be well positioned on the screen independently of the value. Do you see a better approach?

Comment (Wendlereis) [reply src/pages/Lobby/components/PlayersTable/PlayersTable.tsx:14]: very interesting, seems fair

**Review by Wendlereis — APPROVED** [review]

**Pull request merged**